tests/yam/agama/agama.pm

```diff
@@ -25,6 +25,7 @@ sub run {
     my $test = get_required_var('AGAMA_TEST');
     my $reboot_page = $testapi::distri->get_reboot_page();
 
+    sleep(300);
```

Comment:
let's not forget this one after wip is removed ;)

Reply:
e45daadf3